man/systemd.system-credentials.xml

```diff
@@ -471,6 +471,37 @@
           <xi:include href="version-info.xml" xpointer="v257"/>
         </listitem>
       </varlistentry>
+
+      <varlistentry>
+        <term><varname>userdb.user.*</varname></term>
+        <term><varname>userdb.group.*</varname></term>
```

Comment:
i think it would probably be wise to rename these to userdb.transient.user.* 

after all i think most people probably don't think users as being something that possibly could be transient. hence we should put that in the name, to emphasize this, and also keep the door open so that eventually we add a non-transient versions, that write to /etc/ persistently.

Reply:
sounds ok to me too